cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/instantsearch-config.js

```diff
@@ -18,7 +18,7 @@ function enableInstantSearch(config) {
     var hierarchicalMenuValue = {};
     if (config.categoryDisplayNamePath && config.categoryDisplayNamePath.indexOf('New Arrivals') > -1) {
         hierarchicalMenuValue = {
-            "CATEGORIES_NEW_ARRIVALS.level_0": (config.categoryDisplayNamePath || '').split(config.categoryDisplayNamePathSeparator),
+            "newArrivalsCategory.level_0": (config.categoryDisplayNamePath || '').split(config.categoryDisplayNamePathSeparator),
```

Comment:
Those hierarchicalFacets uses `level_0`, `level_1` to identify the level while `__primary_category` uses `0`, `1`. It feels like a good occasion to align the naming.